pkcs.go

```diff
@@ -64,6 +69,9 @@ func PKCSPad(buf []byte, size int) ([]byte, error) {
 func PKCSUnpad(buf []byte, size int) ([]byte, error) {
 	df := DbgOn()
 	defer DbgRestore(df)
+	var (
+		blocks [maxBlocks]Block
+	)
 	if size > 255 {
```

Comment:
That's not really needed, I mean it's equivalent with a make() of fixed size. Using make() does not mean that it would be allocated on the heap. It can still be allocated on the stack (think alloca()), it depends in both cases on the escape analysis. If the compiler decides that the value will be used outside the function (e.g. returned or passed as parameter to another non-inlined and non.trivial function) it would go on the heap, else on the stack. SInce in this case blocks is
 local only (not passed as parameter to anything) it should be ok (well maybe if > 65k it would go on the heap).